p2panda-rs/src/schema/schema.rs

```diff
@@ -15,9 +15,13 @@ type FieldKey = String;
 /// It is constructed from a `SchemaView` and all related `SchemaFieldView`s.
 #[derive(Debug, PartialEq)]
 pub struct Schema {
-    id: DocumentViewId,
-    name: String,
```

Comment:
I'm sort of feeling that it's actually neater to keep these values on the struct.... I know they technically aren't needed, but it saves some mental complexity introduced in the methods. We could then have a getter for the schema id which combined the name and document view.

Reply:
I'm happy to move forward as we are as well though...

Reply:
I see what you mean Sam, it's a good example how bringing concepts closer together does not necessarily mean it's "better", I would say it's more of an compromise sometimes (redundancy gives flexibility and easier to read code but is not DRY, optimization is elegant but less flexible and sometimes leads to loads of nesting).

In that case I'm fine with both how it used to be and the new update, it looks neat now ✨